src/Components/Scenes/Governance/GovernorsTable.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FunctionComponent } from 'react';
+import TableWrapper from '../../Presentational/Table/TableWrapper';
+import MockGovernors from '../../../State/__mockData__/mockGovernors.json';
+
+const GovernorsTable: FunctionComponent = (props: any) => {
```

Comment:
More of a standards thing, I didn't use FunctionComponent because I feel it's unnecessary and just adds more unnecessary code. I can't think of what adding the type FunctionComponent helps but we can talk about this later

Reply:
Sure removed